embulk-core/src/main/java/org/embulk/spi/ExecSession.java

```diff
@@ -219,6 +233,16 @@ public boolean isPreview()
 
     public void cleanup()
     {
+        for (final Reporter.Channel channel : Reporter.Channel.values()) {
+            final AbstractReporterImpl reporter = (AbstractReporterImpl) reporters.get(channel);
+            try {
+                reporter.cleanup();
+            }
+            catch (Exception ex) {
+                // ignore exception
```

Comment:
Do we just ignore? Thought we may want some messages or else. (And, null-check before using.)